### ISSUE TITLE
Move away from abandoned dependencies

### ISSUE THREAD

**Issue by tomudding** [GEWIS/gewisdb#443]
### Current behaviour

> Package laminas/laminas-crypt is abandoned, you should avoid using it. No replacement was suggested.
> Package laminas/laminas-mail is abandoned, you should avoid using it. Use symfony/mailer instead.

### Desired behaviour

`laminas/laminas-mail` is replaced with `symfony/mailer` and `laminas/laminas-crypt` is removed and the existing code is changed to remove its functionality (see https://github.com/search?q=repo%3AGEWIS%2Fgewisdb+%22Laminas%5CCrypt%22&type=code)

### Steps to reproduce

N/A

### Database version

chore/dependencies-2024-10

**Comment by tomudding** [comment]
This should be done before we start using Composer 2.7, as at that point, the behaviour of accepting/reporting abandoned packages will change.

**Comment by tomudding** [comment]
Superseded by #555. This issue has been the umbrella for moving off unmaintained dependencies, and the answer turned out to be the framework itself rather than the packages around it: GEWISDB is migrating from Laminas MVC to Symfony 8.1 on PHP 8.5, which removes every `laminas/*` dependency including the vendored forks. #562 tracks the individual component substitutions. Closing in favour of the epic.
